File: ws/andrade/u3/Ws28/Taxes/src/main/java/edu/ec/espe/taxes/controller/Tax.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package edu.ec.espe.taxes.controller;

/**
 *
 * @author LABS-ESPE
 */
public class Tax {
    public float computeTax(float basePrice, float percentage, boolean charged){
        float ivaValue;
        if (!charged) {
            ivaValue = 0.0F;
        }else {
        ivaValue = basePrice * percentage/100;
    }   
        return ivaValue;
    }
    public float computeIce(float basePrice){
        return 0;
    }
}

File: hw/palacios/u3/HW16 UnitTest/src/controller/Tax.js
class Tax {
    computeTax(basePrice, percentage, charged) {
        if (!charged) {
            return 0.0;
        }
        return basePrice * percentage / 100;
    }

    computeIce(basePrice) {
        // Placeholder para ICE (similar a Java)
        return 0.0;
    }
}

module.exports = Tax;